src/ring_jwt_middleware/core.clj

```diff
@@ -333,9 +333,59 @@
              ['_ `(check-identity-filter! ~authorized (:identity ~'+compojure-api-request+))]))
 
 
+(defn to-scope-repr
+  [txt]
+  (let [[path access] (str/split txt #":")]
+    {:path (str/split path #"/")
+     :access (case access
+               "read"  #{:read}
+               "write" #{:write}
```

Comment:
How is it possible to have write access without read access? I suggest adding `:read` here.

Reply:
The real reason to separate the two notions is for API for example for which an user can push some data (like report a pb) but we don't want him to be able to list it.

~~~
POST /report/:info => report:write
GET /reports => report:read
~~~